web/client/components/map/openlayers/VectorStyle.js

```diff
@@ -113,6 +113,7 @@ export const getTextStyle = (style = {}, stroke = null, fill = null, feature) =>
     return isTextStyle(style) ? new Text({
         fill,
         offsetY: style.offsetY || -( 4 * Math.sqrt(style.fontSize)), // TODO improve this for high font values > 100px
+        rotation: style.textRotation,
```

Comment:
here i would derive the value from textRotationDeg and remove textRotation from the style object